### DISCUSSION TITLE
Windows 10 Start menu opens on wrong side...

### DISCUSSION THREAD

**Discussion by Davikar** [valinet/ExplorerPatcher#964] [General]
I have my task bar to the left.
At first it works as it should, but after switching screens back and forth the start menu now opens on the bottom left (the bottom of the menu is touching the bottom left) instead of the top left. Only way to get it back to normal is to restart it.

**Comment by valinet** [comment]
What does “switching screens back and forth” actually mean?

Can you provide a way to reproduce and a more detailed description? I don’t really understand what is happening.

**Reply by valinet** [reply]
Also, specify what settings you use. Thanks.

**Reply by Davikar** [reply]
Most of the settings are the defaults. Taskbar and start menu style is Windows 10. On the first display setup the primary monitor is a 3440x1440 screen with the taskbar on the left and the secondary monitor is a 1200x1920 screen (flipped 90°) with the taskbar on the bottom. Then I switch everything over to a 4k screen (tv) and the menu is still on the right place.
But then when I switch back to the first setup the start menu is suddenly opening to the bottom left on the primary monitor.

**Reply by valinet** [reply]
Okay, I understand. Test to see how the Windows 11 menu behaves. And yeah, I mean, it’s kind of expected, the Windows 10 menu is barely supported, Microsoft abandoned it.

**Reply by Davikar** [reply]
Yeah, I get that. Also, this is kind of a minor issue, but I thought I'd report it anyways.
